tests/Microsoft.Graph.Core.Test/Microsoft.Graph.Core.Test.csproj

```diff
@@ -16,6 +19,8 @@
     <ReferencePath>$(ProgramFiles)\Common Files\microsoft shared\VSTT\$(VisualStudioVersion)\UITestExtensionPackages</ReferencePath>
     <IsCodedUITest>False</IsCodedUITest>
     <TestProjectType>UnitTest</TestProjectType>
```

Comment:
NuGetPackageImportStamp - remove. 
https://stackoverflow.com/questions/24049961/what-is-nugetpackageimportstamp-for